python/ray/tests/test_state_api.py

```diff
@@ -1741,42 +1740,52 @@ def verify():
         tasks = list_tasks()
         # Actor.__init__: 1 finished
         # Actor.call: 1 running, 9 waiting for execution (queued).
-        correct_num_tasks = len(tasks) == 11
-        waiting_for_execution = len(
-            list(
-                filter(
-                    lambda task: task["scheduling_state"] == "WAITING_FOR_EXECUTION",
-                    tasks,
+        assert len(tasks) == 11
```

Comment:
oh - this wasn't really relevant. But this style allows to see which assertion error happens.